control-plane/cmd/kafka-controller/main.go

```diff
@@ -33,7 +33,7 @@ import (
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/config"
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/kafka/clientpool"
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/broker"
-	"knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/channel"
+	channelv2 "knative.dev/eventing-kafka-broker/control-plane/pkg/reconciler/channel/v2"
```

Comment:
Ah yes, I was just thinking of doing that in a separate PR so the diff here was easier to understand. Do you think it would be better to do both in this one @pierDipi ?

Reply:
fine either way